utils/api/requests.js

```diff
@@ -72,3 +72,100 @@ export const sendMessage = ({ id, message, files }) => {
 
   posting(`/quote_groups/${id}/notes.json`, note)
 }
+
+export const useInitializeRequest = (id) => {
+  const { data, error } = useSWR(`/wares/${id}/quote_groups.json`, fetcher)
+  let dynamicForm = { name: data?.name }
```

Comment:
some wares may not have a dynamic form. we at minimum need the name of the ware though.